DemoApp-ObjC/DemoApp-ObjC/Ads/ALFrameLayoutMrecAdViewController.m

```diff
@@ -0,0 +1,87 @@
+//
+//  ALFrameLayoutMrecAdViewController.m
+//  DemoApp-ObjC
+//
+//  Created by Andrew Tian on 1/23/20.
+//  Copyright © 2020 AppLovin Corporation. All rights reserved.
+//
+
+#import "ALFrameLayoutMrecAdViewController.h"
+#import <AppLovinSDK/AppLovinSDK.h>
+
+@interface ALFrameLayoutMrecAdViewController()<MAAdViewAdDelegate>
+@property (nonatomic, strong) MAAdView *adView;
+@end
+
+@implementation ALFrameLayoutMrecAdViewController
+
+#pragma mark - View Lifecycle
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    self.adView = [[MAAdView alloc] initWithAdUnitIdentifier: @"YOUR_AD_UNIT_ID" adFormat: MAAdFormat.mrec];
+    self.adView.delegate = self;
+    
+    // Dimensions
+    CGFloat width = 300;
+    CGFloat height = 250;
+    CGFloat y = 0;
+    
+    self.adView.frame = CGRectMake(self.view.center.x, y, width, height);
```

Comment:
This is different from the Swift implementation, but I think the Swift one is right.

Reply:
Actually for some reason we can't directly set the `center.x` property in objc so I had to do this way for objc. Lmk if there's a better way